packages/create-plugin/src/cli.ts

```diff
@@ -24,6 +29,29 @@ create('create-plugin', {
   //     prompt: 'if-no-arg',
   //   },
   // },
-  // after: ({ answers }) => console.log(`Ok you chose ${answers.architecture}.`),
+  after: ({ packageDir, template}) => {
+      if (template == "react-custom-dashboard") {
```

Comment:
Could you add a check for the `svelte-custom-dashboard` template as well?

Reply:
Done